plugins/modules/zos_fetch.py

```diff
@@ -26,8 +26,9 @@
     module to copy files from local machine to the remote z/OS system.
   - When fetching a sequential data set, the destination file name will be the
     same as the data set name.
-  - When fetching a PDS/PDS(E), the destination will be a directory with the
-    same name as the PDS/PDS(E).
+  - When fetching a PDS or PDS(E), the destination will be a directory with the
+    same name as the PDS or PDS(E).
+    # why are we using different methods to refer to PDS(E) - elsewhere I see PDSE. It will be helpful if we stick to either one.
```

Comment:
It should be PDSE

Reply:
I believe people think of PDS(members) and augment this in their mind to differentiate between PDS and PDSE where PDS(E) maps to members, but should be PDSE.